ssh/tailssh/incubator.go

```diff
@@ -1014,10 +1014,10 @@ func (ss *sshSession) startWithStdPipes() (err error) {
 
 func envForUser(u *userMeta) []string {
 	return []string{
-		fmt.Sprintf("SHELL=" + u.LoginShell()),
-		fmt.Sprintf("USER=" + u.Username),
-		fmt.Sprintf("HOME=" + u.HomeDir),
-		fmt.Sprintf("PATH=" + defaultPathForUser(&u.User)),
+		fmt.Sprintf("SHELL=%v", u.LoginShell()),
+		fmt.Sprintf("USER=%v", u.Username),
+		fmt.Sprintf("HOME=%v", u.HomeDir),
```

Comment:
I left the printf on these, thinking that we may want to use `%q` at some point, in case any of these values have spaces.  I didn't do that proactively since I'm not 100% sure quoted strings are supported here, and didn't take the time to investigate further.

Reply:
I don't think they're quoted at this level.

Sprintf is fine, or concat. Your call. I might do %s for extra clarity that we know they're strings. But %v is also fine.

Reply:
done (switched to `%s`)